src/video/windows/SDL_windowswindow.c

```diff
@@ -776,18 +776,7 @@ WIN_GetWindowBordersSize(_THIS, SDL_Window * window, int *top, int *left, int *b
 void
 WIN_GetWindowDrawableSize(_THIS, SDL_Window * window, int *w, int *h)
 {
-    HWND hwnd = ((SDL_WindowData *) window->driverdata)->hwnd;
-    RECT rcClient;
-
-    GetClientRect(hwnd, &rcClient);
-
-	if (w) {
-    	*w = rcClient.right - rcClient.left;
-	}
-
-	if (h) {
-    	*h = rcClient.bottom - rcClient.top;
-	}
+    WIN_GetDrawableSize(window, w, h);
```

Comment:
Yeah, the D3D drivers should call the public function rather than calling `WIN_GetDrawableSize()` directly.

Reply:
As in they should call the new public SDL_GetWindowDrawableSize?

Reply:
Made it so that all the Windows OpenGL/GLES/D3D/etc functions call the public API, merged `WIN_GetDrawableSize` into the main driver function for `SDL_GetWindowSizeInPixels`